packages/create-canon/scaffold/app/pages/Home.jsx

```diff
@@ -12,3 +12,5 @@ export default class Home extends Component {
   }
 
 }
+
+export default Home
```

Comment:
does this pass the eslint? missing an ending semicolon? 😏

Reply:
:pikachu: